build/run_local_server.sh

```diff
@@ -32,4 +32,7 @@ fi
 # Kill any existing instances
 pkill -f appflowy_cloud || true
 
+# Run the migrations
+cargo sqlx database create && cargo sqlx migrate run && cargo sqlx prepare
```

Comment:
when performing docker compose. The migration should auto run by this line of code:

`- ./migrations:/docker-entrypoint-initdb.d`

So, is it necessary to create database and run the migrations?

Reply:
let's say you change some SQL file, the schema of DB will change, but it will not update the .sqlx files

Reply:
Got it.

So, it may just need to run : `cargo sqlx prepare`